src/placeholder.ts

```diff
@@ -70,9 +81,20 @@ export class Placeholder<A> extends Behavior<A> {
   }
 }
 
-class MapPlaceholder<A, B> extends MapBehavior<A, B> {}
+class MapPlaceholder<A, B> extends MapBehavior<A, B> {
+  pushS(t: number, a: A) {
+    // @ts-ignore
```

Comment:
Why ts-ignore?

Reply:
could be solved using casting, but it would make it less readable